app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -762,16 +763,32 @@ class PipelineSampleReport extends React.Component {
     return foo;
   }
 
-  render_number(x, emphasize, num_decimals) {
-    const is_blank = (x == 0) || (x == -100);
-    let y = Number(x);
-    y = y.toFixed(num_decimals);
-    y = numberWithCommas(y);
-    if (emphasize) {
-      y = <b>{y}</b>;
-    }
-    const className = is_blank ? 'report-number-blank' : 'report-number';
-    return (<td className={className}>{y}</td>);
+  render_number(ntCount, nrCount, num_decimals, isAggregate = false) {
```

Comment:
This function has a lot of repeated code in it. Does it only exist to distinguish the aggregate score? Why not just use `state.countType` instead of NT/NR in lines 1187-1192, while leaving line 1186 as tax_info.NT.aggregatescore?

Reply:
No, this is used to render both NT and NR count under a single column, it also ascertain whether or not the NT or NR value is blank, or when you switch focus on the NT or NR

Reply:
I see, cool. Can you refactor with an additional function so that you don't have the duplicated lines for NT and NR?